app/models/concerns/compute_scaling.rb

```diff
@@ -4,11 +4,11 @@ module ComputeScaling
 
   # GCE machine types and file size ranges
   # produces a hash with entries like { 'n2-highmem-4' => 0..16.gigabytes }
-  # change GB_PER_CORE in parent class to adjust scaling (1-8, higher number means more aggressive scaling)
+  # change GB_PER_CORE in parent class to adjust scaling (1-8, lower number means more aggressive scaling)
```

Comment:
"More aggressive scaling" means "more RAM per core", right?  If so, higher (not lower) number means more aggressive scaling.
```suggestion
  # change GB_PER_CORE in parent class to adjust scaling (1-8, higher number means more aggressive scaling)
```

Reply:
No, I originally thought the same but realized it really is the inverse that is true, and I think because the name is misleading.  The GB_PER_CORE number is used as the multiplier when determining ranges for machine types, _not_ the actual number of GB being allocated per core.  The latter is constant at 8 GB.  Instead, setting GB_PER_CORE lower causes the ranges between machines to be smaller, meaning that as file size increases, the machine type scales more rapidly.

I think I'm going to change the name of that constant to be less confusing.

Reply:
[51b193a](https://github.com/broadinstitute/single_cell_portal_core/pull/2154/commits/51b193a17318ef9358042388b3f7c0fb49ce9797)